x-pack/plugins/apm/public/components/shared/charts/spark_plot/index.tsx

```diff
@@ -21,7 +21,7 @@ import { useChartTheme } from '../../../../../../observability/public';
 import { px, unit } from '../../../../style/variables';
 import { useTheme } from '../../../../hooks/use_theme';
 
-type Color =
+export type Color =
```

Comment:
Can we rename this to something like SparkplotEuiColorName?

Reply:
This no longer needs to be exported I removed the ComparisonSparkPlot